app/services/rapid_ready_for_decision/lighthouse_medication_request_data.rb

```diff
@@ -2,39 +2,36 @@
 
 module RapidReadyForDecision
   class LighthouseMedicationRequestData
-    attr_accessor :response
-
     def initialize(response)
       @response = response
     end
 
-    def transform
-      transformed_entries = filtered_entries.map { |entry| transform_entry(entry) }
-      sorted_entries(transformed_entries)
+    def count
+      # Should be the same as @response.body["total"]
+      resources.size
     end
 
-    private
-
-    def sorted_entries(transformed_entries)
-      transformed_entries.sort_by { |med| [med[:authoredOn].to_datetime, med[:description]] }.reverse!
+    def resources
+      @resources ||= @response.blank? ? [] : @response.body['entry'].map { |mr| mr['resource'] }
     end
 
-    def filtered_entries
-      response.body['entry'].filter { |entry| entry['resource']['status'] == 'active' }
+    def transform
+      active_med_requests.map { |resource| MedicationEntry.new(resource).result }
+                         .sort_by { |med| [med[:authoredOn].to_datetime, med[:description]] }
+                         .reverse!
     end
 
-    def transform_entry(raw_entry)
-      entry = raw_entry['resource'].slice(
-        'status', 'medicationReference', 'subject', 'authoredOn', 'note', 'dosageInstruction'
-      )
```

Comment:
`slice` call is unnecessary since `MedicationEntry#result` calls `slice`. Removing this line leaves only line 31 for `transform_entry`, whose method name doesn't really mean much, so I got rid of the method and called `MedicationEntry.new(entry).result` directly.